### ISSUE TITLE
Disabling alt text indicators doesn't work

### ISSUE THREAD

**Issue by BluePixel4k** [LucasGGamerM/moshidon#61]
**Describe the bug**
I tried to disabling the both alt text indicators, but it doesn't work. After a reboot of the app, the settings is set to their defaults.

**To Reproduce**
1. Go to 'Settings'
2. Click on the toggles for 'Indicator for alt texts' and 'Indicator for missing alt texts'
3. Go to the home feed
4. Reboot the app
5. The both toggles have been reset to their default values

**Screenshots and screen recordings**
<img src="https://user-images.githubusercontent.com/77979781/215319783-bd2fbcf5-7708-4f7e-8c74-4514d6f46eaf.png" width="300" />

**Version**
Megalodon version: [e.g. v1.1.4+fork.88]
Android 13 (GrapheneOS)

**Comment by LucasGGamerM** [comment]
Fixed in a2a0c9801b56ba120f6f116812f9218d48a61f81
